frontend/packages/kubevirt-plugin/src/constants/vm/storage/disk-type.ts

```diff
@@ -1,11 +1,16 @@
 /* eslint-disable lines-between-class-members */
+import * as _ from 'lodash';
 import { ValueEnum } from '../../value-enum';
 
 export class DiskType extends ValueEnum<string> {
   static readonly DISK = new DiskType('disk');
   static readonly CDROM = new DiskType('cdrom');
   static readonly FLOPPY = new DiskType('floppy');
   static readonly LUN = new DiskType('lun');
+  static readonly typeLabels = {
```

Comment:
this could get confused with the enum: please see https://github.com/openshift/console/pull/3436#pullrequestreview-322294057

Reply:
moved the dictionary away from the consts and renamed it.

Reply:
also added a constractor option :-)